### PR TITLE
Feature endorseuser

### PR DESCRIPTION
**Pull request by brunoliberal**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,6 +7,5 @@ target
 src/main/webapp/lib/*
 src/main/webapp/node/*
 src/main/webapp/node_modules/*
-resources/client_secrets.json
```
Comment (larruda) [inline .gitignore]: Really?

File: src/main/webapp/js/jsUtils.js [hunk 2]
```diff
@@ -98,6 +98,8 @@ var mockShowEndorsementResponse = function(){
         
         response.endorsed.name = names[i-1];
         response.endorsed.photo = "https://storage.googleapis.com/tech-gallery-assets/userPhotos/user" + i + ".jpg";
+        response.endorsed.clientId = i;
+        response.endorsed.email = response.endorsed.name+"@example.com";
```
Comment (larruda) [inline src/main/webapp/js/jsUtils.js]: Is this correct?

File: src/test/java/com/ciandt/techgallery/service/EndorsementServiceTest.java [hunk 3]
```diff
@@ -77,23 +75,23 @@ public void executaAgrupamento() throws BadRequestException, NotFoundException,
   }
 
   private List<Endorsement> createEndorsementList() {
-    //TODO incrementar para um cenário de teste mais completo
+    // TODO incrementar para um cenário de teste mais completo
```
Comment (larruda) [inline src/test/java/com/ciandt/techgallery/service/EndorsementServiceTest.java]: Comments in English, please ;-)

File: .gitignore (hunk 1 above)
Comment (brunoliberal) [inline .gitignore]: The project is not using it anymore, but if you want to i can add it again.

File: src/main/webapp/js/jsUtils.js (hunk 2 above)
Comment (brunoliberal) [inline src/main/webapp/js/jsUtils.js]: It's inside a mock method, just for example. In the future the full method can be removed.

**Pull request merged**